tests/unittests/quantizationTest.cpp

```diff
@@ -154,7 +154,7 @@ createSimpleGraphForQuantization(Module *M) {
                                            Variable::TrainKind::None);
   fillStableRandomData(B->getHandle(), 2001, 1);
 
-  ConvolutionNode *CV = F->createConv("conv", A, 16, 5, 1, 2, 1);
```

Comment:
Why did you change this test?

Reply:
To test group arg > 1. Since Input channels should be divisible by group, I modified both of them to 2.